.aspect/workflows/config.yaml

```diff
@@ -5,6 +5,8 @@ tasks:
       queue: aspect-small-amd64
   - buildifier:
       queue: aspect-small-arm64
+  # - configure:
```

Comment:
Configure not working on rc.9 on CCI

Reply:
What was the issue?

Reply:
The Setup step blows up. We don't have coverage for that on silo since we vendor in the CCI yaml.

Reply:
Here's a build with the issue: https://app.circleci.com/pipelines/github/aspect-build/aspect-cli/1287/workflows/e5e4064b-4f92-446d-93bb-9ff4aee78644/jobs/3170